pipeline/acquisition.py

```diff
@@ -0,0 +1,375 @@
+
+
+@schema
+class IntracellularInfo(dj.Manual):
+    definition = """ # Table containing information relating to the intracelluar recording (e.g. cell info)
+    -> Session
+    cell_id: varchar(36) # a string identifying the cell in which this intracellular recording is concerning
+    ---
+    cell_type: enum('excitatory','inhibitory','N/A')
+    -> ActionLocation
+    -> reference.Device
+    """    
+    
+    
+@schema
+class ExtracellularInfo(dj.Manual):
+    definition = """ # Table containing information relating to the extracelluar recording (e.g. location)
+    -> Session
+    ec_id: varchar(36) # extracellular_id: a string uniquely identify this extracellular recording routine of this recording session (there might be multiple extracellular recording in 1 recording session, at different location for example)
+    ---
+    -> ActionLocation
+    -> reference.Device
+    """    
+    
+    
+@schema
+class StimulationInfo(dj.Manual):
+    definition = """ # Table containing information relating to the stimulatiom (stimulation type (optical or electrical), location, device)
+    -> Session
+    stim_id: varchar(36) # stimulation_id: a string uniquely identify this stimulation routine of this recording session (there might be multiple stimulation of the same or different type (phototim or electrical) in 1 recording session)
+    ---
+    stim_type: enum('optical','electrical')
+    -> ActionLocation
+    -> reference.Device
+    """    
+
+
+@schema
+class TrialSet(dj.Imported):
+    definition = """
+    -> Session
+    ---
+    n_trials: int # total number of trials
+    """
+    
+    class Trial(dj.Part):
+        definition = """
+        -> master
+        trial_id: varchar(36)           # unique id of this trial in this trial set
+        ---
+        cue_start_time: float           # cue onset of this trial, with respect to this trial's start time
+        cue_end_time: float             # cue end of this trial, with respect to this trial's start time
+        pole_in_time: float             # the start of sample period for each trial (e.g. the onset of pole motion towards the exploration area), relative to trial start time
+        pole_out_time: float            # the end of the sample period (e.g. the onset of pole motion away from the exploration area), relative to trial start time
+        start_time: float               # start time of this trial, with respect to starting point of this session
+        stop_time: float                # end time of this trial, with respect to starting point of this session
+        """
+        
+    class TrialInfo(dj.Part):
+        definition = """
+        -> master.Trial
+        ---
+        trial_type: enum('Lick L trial','Lick R trial') # the experimental type of this trial, e.g. Lick Left vs Lick Right
+        trial_stim_status: enum('Stim','No stim') # is this trial a Stimulation or No stimulation trial
+        trial_status: enum('good','bad') # the good/bad status of this trial
```

Comment:
Similarly this sounds like it would be simpler to use `good_trial: bool`. If you anticipate that this can increase in more values (more than good and bad), then break this out into a lookup table)